README.md

```diff
@@ -1,11 +1,11 @@
-# algorand-builder-templates
+# algo-builder-templates
 
-Dapps templates for [Algorand Builder](https://github.com/scale-it/algorand-builder). Use these templates to quickly setup/get started with your Dapp implementation.
+Dapps templates for [Algorand Builder](https://github.com/scale-it/algo-builder). Use these templates to quickly setup/get started with your Dapp implementation.
 
 ## Usage
 Each template is a single project (with 1 package.json file) comprising of the following:
- - An `algob` project : Used to deploy ASA's, stateless and stateful contracts using algob `deployer` object. Use `algob deploy` to deploy your `/scripts`. Sample algob project can  be found [here](https://github.com/scale-it/algorand-builder/tree/master/packages/algob/sample-project), check the project readme.md for more details.
-   Read more about `@algorand-builder/algob` [here](https://github.com/scale-it/algorand-builder/tree/master/packages/algob).
+ - An `algob` project : Used to deploy ASA's, stateless and stateful contracts using algob `deployer` object. Use `algob deploy` to deploy your `/scripts`. Sample algob project can  be found [here](https://github.com/scale-it/algo-builder/tree/master/packages/algob/sample-project), check the project readme.md for more details.
+   Read more about `@algo-builder/algob` [here](https://github.com/scale-it/algo-builder/tree/master/packages/algob).
```

Comment:
Maybe we should link to the root? What do you think?

Reply:
Yes, good idea.